ibm/service/schematics/resource_ibm_schematics_workspace.go

```diff
@@ -470,7 +470,7 @@ func ResourceIBMSchematicsWorkspaceValidator() *validate.ResourceValidator {
 			Identifier:                 schematicsWorkspaceTemplateType,
 			ValidateFunctionIdentifier: validate.ValidateRegexp,
 			Type:                       validate.TypeString,
-			Regexp:                     `^terraform_v(?:0\.11|0\.12|0\.13|0\.14|0\.15|1\.0)(?:\.\d+)?$`,
+			Regexp:                     `^terraform_v(?:0\.11|0\.12|0\.13|0\.14|0\.15|1\.0|1\.1|1\.2)(?:\.\d+)?$`,
```

Comment:
1.2 we don't yet support in Schematics its planned in August

Reply:
Can you check repsective documentaion also should we add the supported verison their

Reply:
sure. will remove 1.2.